Misc/NEWS.d/next/Tests/2019-04-21-17-53-50.bpo-32424.Q4rBmn.rst

```diff
@@ -0,0 +1,2 @@
+Fix typo in test_cyclic_gc() test for xml.etree.ElementTree. Patch by Gordon
```

Comment:
Phew, is this really worth a separate news entry? I think it's pretty much covered by the other one.

Reply:
It's a separate commit that fixes a bug in an existing test rather than adding any new ones. That to me is a distinct change from "improving test coverage"—which I intended to refer literally to test coverage percentage.